mopro-core/src/middleware/circom/serialization.rs

```diff
@@ -37,16 +39,22 @@ pub fn deserialize_proving_key(data: Vec<u8>) -> SerializableProvingKey {
         .expect("Deserialization failed")
 }
 
-pub fn serialize_inputs(inputs: &SerializableInputs) -> Vec<u8> {
-    let mut serialized_data = Vec::new();
-    inputs
-        .serialize_uncompressed(&mut serialized_data)
-        .expect("Serialization failed");
+pub fn serialize_inputs(inputs: &SerializableInputs) -> Vec<String> {
+    let serialized_bigint_data: Vec<BigUint> =
```

Comment:
I wonder if this function can be written more in same style as old one? Not sure diff has to be this big with this many operations (could be wrong though)